[2023-10-17T11:19:33.400+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [queued]>
[2023-10-17T11:19:33.407+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [queued]>
[2023-10-17T11:19:33.408+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:33.418+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-07 00:00:00+00:00
[2023-10-17T11:19:33.423+0000] {standard_task_runner.py:57} INFO - Started process 685 to run task
[2023-10-17T11:19:33.425+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-07T00:00:00+00:00', '--job-id', '131', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpc6mw9sr0']
[2023-10-17T11:19:33.425+0000] {standard_task_runner.py:85} INFO - Job 131: Subtask waiting_for_data
[2023-10-17T11:19:33.463+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:33.528+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-07T00:00:00+00:00'
[2023-10-17T11:19:33.535+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:33.536+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:33.536+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:33.536+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:33.543+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210507T000000, start_date=20231017T111933, end_date=20231017T111933
[2023-10-17T11:19:33.556+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:33.568+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:43:07.360+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [queued]>
[2023-10-18T10:43:07.368+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [queued]>
[2023-10-18T10:43:07.368+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:43:07.385+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-07 00:00:00+00:00
[2023-10-18T10:43:07.393+0000] {standard_task_runner.py:57} INFO - Started process 1648 to run task
[2023-10-18T10:43:07.397+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-07T00:00:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpmeahopar']
[2023-10-18T10:43:07.398+0000] {standard_task_runner.py:85} INFO - Job 130: Subtask waiting_for_data
[2023-10-18T10:43:07.453+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:43:07.532+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-07T00:00:00+00:00'
[2023-10-18T10:43:07.547+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:43:07.550+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210507T000000, start_date=20231018T104307, end_date=20231018T104307
[2023-10-18T10:43:07.560+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 130 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1648)
[2023-10-18T10:43:07.609+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:43:07.639+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:43.300+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [queued]>
[2023-10-18T11:17:43.309+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [queued]>
[2023-10-18T11:17:43.309+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:43.320+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-07 00:00:00+00:00
[2023-10-18T11:17:43.327+0000] {standard_task_runner.py:57} INFO - Started process 569 to run task
[2023-10-18T11:17:43.329+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-07T00:00:00+00:00', '--job-id', '216', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpjh191zey']
[2023-10-18T11:17:43.330+0000] {standard_task_runner.py:85} INFO - Job 216: Subtask waiting_for_data
[2023-10-18T11:17:43.371+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:43.442+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-07T00:00:00+00:00'
[2023-10-18T11:17:43.452+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:43.453+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:43.454+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:43.454+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:43.463+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210507T000000, start_date=20231018T111743, end_date=20231018T111743
[2023-10-18T11:17:43.501+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:43.529+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:43:50.828+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [queued]>
[2023-10-19T12:43:50.839+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [queued]>
[2023-10-19T12:43:50.840+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:43:50.851+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-07 00:00:00+00:00
[2023-10-19T12:43:50.857+0000] {standard_task_runner.py:57} INFO - Started process 1168 to run task
[2023-10-19T12:43:50.859+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-07T00:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpmzllao4x']
[2023-10-19T12:43:50.860+0000] {standard_task_runner.py:85} INFO - Job 252: Subtask waiting_for_data
[2023-10-19T12:43:50.898+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-07T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:43:50.965+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-07T00:00:00+00:00'
[2023-10-19T12:43:50.973+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:43:50.974+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:43:50.974+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:43:50.975+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:43:50.981+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210507T000000, start_date=20231019T124350, end_date=20231019T124350
[2023-10-19T12:43:51.031+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:43:51.055+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
